end-to-end-tests/tst/helpers/base_test.py

```diff
@@ -0,0 +1,86 @@
+import unittest
+import boto3
+import random
+import os
+import subprocess
+from resources import Resources
+
+
+class BaseTest(unittest.TestCase):
+    def setUp(self):
+        self.stack_name = "sam-temp-stack" + str(random.randint(0, 10000))
```

Comment:
instead of `random.randint` you should use `uuid`

Reply:
thank you, updated the code with `uuid`